blobstore/src/main/java/org/jclouds/blobstore/config/LocalBlobStore.java

```diff
@@ -100,6 +100,7 @@
 
 @Singleton
 public final class LocalBlobStore implements BlobStore {
+   private static final String MULTIPART_PREFIX = ".mpus-";
```

Comment:
I don't think so; it might be nice to configure MPUs to go into another directory or similar but I don't see much need to change the prefix.